src/modules/processes.nix

```diff
@@ -35,7 +35,7 @@ let
   implementation-options = config.process.${implementation};
   envList =
     lib.mapAttrsToList
-      (name: value: "${name}=${builtins.toJSON value}")
+      (name: value: "${name}=${toString value}")
```

Comment:
Just ran into this myself. Please choose the function based on the implementation (process-compose=> toString else `builtins.toJSON` as it was.)

Then this fixes the currently broken escaping. @domenkozar this fixes the completely broken process-compose implementation since the breaking change of https://github.com/cachix/devenv/commit/8bc83de411910ec5ed9759f8478c49639326e5f0`